src/applications/combined-debt-portal/combined/utils/helpers.js

```diff
@@ -116,3 +116,7 @@ export const setPageFocus = selector => {
     document.querySelector('#main h1').focus();
   }
 };
+
+export const setDocumentTitle = title => {
```

Comment:
Since this is only used in the `useHeaderPageTitle` I think we can just move the code to that file so it's obfuscated a little less and easier to read. Not blocking, just a suggestion, otherwise looks solid!